### ISSUE TITLE
[FEATURE]-System Monitor - Expose program reset, processor reboot, and appliance/server feedback (SIMPL Bridging)

### ISSUE THREAD

**Issue by jkdevito** [PepperDash/Essentials#1053]
**Describe the solution you'd like**
Add support to the System Monitor class and bridge for the following 
1. Program reset
- Ideally implemented with a press&hold 
2. Processor reboot
3. Feedback on whether the code is running on a server vs. an appliance

**Additional context**
We must consider how these updates may also affect a server and whether they would add benefit or need protection when Essentials runs on a server.

**Comment by ndorin** [comment]
@jkdevito Can you confirm our assumption that this feature would enhance SIMPL bridging only, as the rest of this is accessible natively in S# Pro?

**Comment by jkdevito** [comment]
Correct, this would be a feature to expose the program reset, processor reboot, and appliance vs. server FB on the system monitor EISC in SIMPL, allowing developers to maintain a consistent implementation regardless of the processor type defined.

**Comment by jtalborough** [comment]
@jkdevito. just checking. Any movement on this by chance?

**Comment by jkdevito** [comment]
It appears to have been merged in on [PR #1070 ](https://github.com/PepperDash/Essentials/pull/1070).  If that is correct we can close this feature request.

**Comment by jkdevito** [comment]
Looking closer at the PR, it appears to only have been merged into `development`.
